justin-h-im/about page

justin-h-im opened the pull request:
Created basis for the About Us title page. 
Images for Code Changes: 
![image](https://github.com/acm-ucr/witl-website/assets/93964548/1515690a-d044-43f0-a2ef-fe74861cdb99)
![image](https://github.com/acm-ucr/witl-website/assets/93964548/a98840f6-5bca-4448-8b30-d58a78e83cae)

Images for Product on Local Host:
![image](https://github.com/acm-ucr/witl-website/assets/93964548/d6cf5ce2-900c-4914-9f6d-f7685f303502)

justin-h-im committed:
Corrected spacing

justin-h-im committed:
Proper version, cleared with prettier

justin-h-im committed:
Adjusted font size and spacing

stanleylew5 commented on src/app/about/page.js:
```diff
@@ -1,4 +1,17 @@
-const Page = () => {
-  return <div>Hello World</div>;
+import "@/app/layout.js";
```
```suggestion

```

stanleylew5 commented on src/app/about/page.js:
```diff
@@ -1,4 +1,17 @@
-const Page = () => {
-  return <div>Hello World</div>;
+import "@/app/layout.js";
```
```suggestion

```

stanleylew5 replied:
not needed as layout.js is already applied

stanleylew5 commented on src/app/about/page.js:
```diff
@@ -1,4 +1,17 @@
+  );
 };
-export default Page;
+export default About;
```
change the exporting back to page and move your code to an About.jsx file

stanleylew5 commented on tailwind.config.js:
```diff
@@ -18,6 +21,9 @@ module.exports = {
         },
       },
     },
+    backgroundColor: (theme) => ({
+      purplecustom: "#2E0144",
+    }),
```
```suggestion
   
```

stanleylew5 replied:
not needed, if you wanted to use this color as a background color use one of the already defined colors with tailwind

the class would be bg-witl-purple-300 (check above for already defined colors)

stanleylew5 commented on src/app/about/page.js:
```diff
@@ -1,4 +1,17 @@
+
+const About = () => {
+  return (
+    <div className="bg-purplecustom min-h-screen flex items-center justify-center">
```
```suggestion
    <div className="bg-witl-purple-300 min-h-screen flex items-center justify-center">
```

stanleylew5 commented on src/app/about/page.js:
```diff
@@ -1,4 +1,17 @@
+  return (
+    <div className="bg-purplecustom min-h-screen flex items-center justify-center">
+      <div className="flex flex-col tracking-widest">
+        <div className="mb-1 text-9xl text-center text-purple-200">
```
use witl colors

stanleylew5 commented on src/app/about/page.js:
```diff
@@ -1,4 +1,17 @@
+        <div className="mb-1 text-9xl text-center text-purple-200">
+          <p>ABOUT</p>
+        </div>
+        <div className="mb-1 text-9xl text-center text-purple-200">
```
use witl colors

justin-h-im committed:
Reformatted about page into a component

justin-h-im committed:
Implemented some code suggestions

justin-h-im committed:
Removed unnecessary line of code

justin-h-im committed:
Merged with dev branch, resolved conflicts

justin-h-im committed:
Corrected colors displayed on the page

justin-h-im commented:
Updated work to put everything into a jsx file, corrected colors, merged with dev on this branch.

Code Screenshots:
![image](https://github.com/acm-ucr/witl-website/assets/93964548/a2c664e6-9144-4415-8736-bfceb57f5935)
![image](https://github.com/acm-ucr/witl-website/assets/93964548/d667291d-1355-4dd9-9540-beac28e3aa3e)

Output Screenshots:
![image](https://github.com/acm-ucr/witl-website/assets/93964548/0be462f2-dcde-48da-a38b-014fdaf2fd20)
